XenAdmin/Dialogs/AttachUsbDialog.cs

```diff
@@ -88,7 +144,7 @@ public HostItem(Host host) :base(false)
 
         private class UsbItem : CustomTreeNode
         {
-            private PUSB _pusb;
+            public PUSB _pusb { get; }

```

Comment:
Please use PascalCasing for public members (capitalizes the first character, no underscore)

Reply:
OK.